app/controllers/tariffs_controller.rb

```diff
@@ -1,44 +1,41 @@
 class TariffsController < ApplicationController
+  def bundle_tariff
+    @tariff = Tariff.find(params[:id])
+  end
   def index
     @tariff = Tariff.all
   end
-
   def new
     @tariff = Tariff.new
   end
-
   def show
-    @tariff = Tariff.find(params[:id])
+    bundle_tariff
   end
-
   def edit
-    @tariff = Tariff.find(params[:id])
+    bundle_tariff
   end
-
   def update
-    @tariff = Tariff.find(params[:id])
+    bundle_tariff
     if @tariff.update(tariff_params)
       redirect_to @tariff
     else render 'edit'
     end
   end
-
   def destroy
-    @tariff = Tariff.find(params[:id])
+    bundle_tariff
     @tariff.destroy
     redirect_to tariffs_path
   end
-
   def create
     # render plaine: params[:post].inspect
     @tariff = Tariff.new(tariff_params)
-
     if @tariff.save
       redirect_to @tariff
     else render 'new'
     end
   end
-private def  tariff_params
+  private
+    def tariff_params
     params.require(:tariff).permit(:name, :price)
```

Comment:
please use rubocop format
`rubocop -a /path/to/your/project`

Reply:
i did not undestend what i have to do in this comment

Reply:
@romigos  https://github.com/rubocop-hq/rubocop